bot/exts/moderation/voice_gate.py

```diff
@@ -37,240 +34,185 @@
 
 VOICE_PING = (
     "Wondering why you can't talk in the voice channels? "
-    f"Use the `{BotConfig.prefix}voiceverify` command in here to verify. "
+    "Click the Voice Verify button in here to verify. "
```

Comment:
```suggestion
    "Click the Voice Verify button above to verify. "
```
May be slightly clearer